src/components/Card/HorizontalCard/HorizontalCard.jsx

```diff
@@ -0,0 +1,92 @@
+import { Rating } from "../../../components";
+import "./HorizontalCard.css";
+export const HorizontalCard = ({ prod, isWishlist, btnTitle, featured }) => {
+	return (
+		<li key={prod._id} className="categories__list">
+			<article
+				className={`card card--shopping card--cart card--horizontal_v2 ${
+					featured ? "card--badge" : ""
+				}`}
+			>
+				<div className="card__content flex">
+					<header className="card__header flex-center">
+						<figure className="card__figure">
+							<img
+								className="card_img image--responsive"
+								src={prod.src}
+								alt={prod.title}
+							/>
+						</figure>
+					</header>
+					<div className="card__body">
+						<div className="card__header_text">
+							<h2 className=" h6">{prod.title}</h2>
+						</div>
+
+						<p className="card__price flex">
+							<span className="disc_price">₹{prod.price.discounted}</span>
+							<span className="original_price">₹{prod.price.original}</span>
+							<span className="pct_disc">
+								{prod.price.percentageDiscount}% off
+							</span>
+						</p>
+
+						<div className="card__quantity_wrapper">
+							<div className="card__quantity form__input_box">
+								<label className="label" for="">
+									Quantity:
+								</label>
+								<select className="card__quantity_values">
+									<option className="input_box" value="1" selected>
```

Comment:
selected will not be hard-coded here

Reply:
I have acknowledged it. Will update it when adding functionality. Thanks for suggesting